test/tx-builder/dydx-governance.spec.ts

```diff
@@ -18,15 +18,20 @@ function init(ctx: TestContext): void {
   txBuilder = new TxBuilder(
     {
       network: Network.hardhat,
-      hardhatGovernanceAddresses: { DYDX_GOVERNANCE: ctx.governor.address } as tDistinctGovernanceAddresses,
+      hardhatGovernanceAddresses: {
+        DYDX_GOVERNANCE: ctx.governor.address,
+      } as tDistinctGovernanceAddresses,
+      hardhatMerkleDistributorAddresses: {
+        MERKLE_DISTRIBUTOR_ADDRESS: ctx.merkleDistributor.address,
+      },
       injectedProvider: hre.ethers.provider,
     },
   );
 }
 
-describeContract('DydxGovernance', init, (ctx: TestContext) => {
+describeContract('DydxGovernance Related Functions', init, (ctx: TestContext) => {
   describeContractForNetwork(
-    'DydxGovernance',
+    'DydxGovernance Related Functions',
```

Comment:
nit: Can you make a separate test file for the merkle distributor? IMO this is separate functionality from the governance service since it's using the merkle distributor service.